### ISSUE TITLE
support for polymer 2

### ISSUE THREAD

**Issue by cordovapolymer** [vguillou/poly-filter#5]
Hi @vguillou, are the plans to [upgrade](https://www.polymer-project.org/2.0/docs/upgrade) the element to support polymer 2, making it hybrid?

**Comment by vguillou** [comment]
Yes and no. I will make a Polymer 2 version, but it will likely not be hybrid. And I can't announce a schedule for that unfortunately... Might come very soon, or in several months.

_Sent from my Motorola XT1562 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by cordovapolymer** [comment]
@vguillou polymer 2 class based version is even better
